redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -257,7 +257,14 @@ public function getCacheFilename()
     {
         assert(null !== $this->cachePath);
         assert(null !== $this->type);
-        return $this->cachePath . $this->type . '/' . $this->originalFilename;
+
+        $cachePath = rex_extension::registerPoint(new rex_extension_point('MEDIA_MANAGER_CACHE_FILENAME_PATH', $this->cachePath, [
+            'type' => $this->type,
+            'filename' => $this->originalFilename,
+            'effects' => $this->effectsFromType($this->getMediaType()),
```

Comment:
Hier werden die Effekte jetzt immer aus der DB abgefragt, auch wenn das Bild aus dem Cache ausgeliefert wird.
Das finde ich suboptimal.

Reply:
Ja, das stimmt. Das macht das Addon aktuell auch so. 
Das nutzt das Addon um festzustellen, ob der negotiator MM Effekt auf das Bild angewendet werden soll oder nicht. 
Gibt es eine andere Möglichkeit um das zu prüfen? Also um alle Effekte zu bekommen, die für dieses Bild angewendet werden?

Reply:
würde es dir reichen wenn ich dir an der Stelle einfach nur $this übergebe?

Reply:
also ein MEDIA_MANAGER_COBJECT

Reply:
Nein, ich glaube das reicht nicht. 
Das media manager Objekt enthält nicht die Effekte dieses Bildes. Aber der Cache Pfad sollte ja nur geändert werden wenn der "Negotiator" Effekt vorhanden ist. 

![image](https://github.com/user-attachments/assets/aaedd705-c774-43e2-946c-22669ed9e57f)

Reply:
@gharlan du kennst dich hier besser aus, fällt dir eine elegante Möglichkeit ein, wie man diesen EP hier am besten umsetzen könnte?

Reply:
@AndiLeni ich meine wenn ich dir nur $this gebe, dann könntest du es doch auslesen?

Reply:
hier geht's weiter: https://github.com/redaxo/redaxo/pull/6135#issuecomment-2374380175